lib/utils/report.mjs

```diff
@@ -47,10 +47,9 @@ export default function report(problem) {
 		);
 	}
 
-	const range = node?.rangeBy({ index, endIndex }) ?? {};
 	// If a line is not passed, use the node.rangeBy method to get the
 	// line number that the complaint pertains to
-	const startLine = line ?? range.start?.line;
+	const startLine = line ?? node?.rangeBy({ index, endIndex }).start.line;
```

Comment:
remark
off-topic

l89 and l56 give us a hint that the first `?` cannot be removed or that `Problem['node']` should be optional.